components/brave_player/resources/playerEmbed.tsx

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { render } from 'react-dom'
+import styled from 'styled-components'
+
+const EmbedContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+`
+
+const StyledFrame = styled.iframe`
+  max-width: 1000px;
+  width: 80%;
+  height: 80%;
+  border: none;
+`
+
+function getFrameSrc () {
+  let paths = new URL(window.location.href).pathname.split('/')
+  if (paths.length < 3) {
+    return ''
+  }
+
+  if (!paths[0]) {
+    // could be empty. Drop it.
```

Comment:
when is it empty?

Reply:
Oh, this shouldn't be "could". The first elem is always empty.

<img width="349" alt="image" src="https://github.com/brave/brave-core/assets/5474642/e6bd1aee-7442-4936-bb95-aa7cd3b22e86">

Let me update code. Maybe we don't have to slice the array and simple destructuring should be fine.